src/main/java/executor/command/CommandGetSpendingByMonth.java

```diff
@@ -26,16 +28,55 @@ public void execute(TaskList taskList) {
 
     @Override
     public void execute(Wallet wallet) {
-        ReceiptTracker receiptsInMonth = new ReceiptTracker();
-        String monthStr = Parser.parseForPrimaryInput(CommandType.EXPENDEDMONTH, userInput);
-        int month = monthStrToInt(monthStr);
-        if (month != 0) {
-            int year = Integer.parseInt(Parser.parseForFlag("year", userInput));
-            receiptsInMonth = wallet.getReceipts().findReceiptByMonthYear(month, year);
-            Double totalMoney = receiptsInMonth.getTotalCashSpent();
-            Ui.dukeSays("The total amount of money spent in " + monthStr + " " + year + " : " + totalMoney);
-        } else {
-            Ui.dukeSays("Invalid input, CORRECT FORMAT : expendedmonth <month> <year> ");
+        try {
```

Comment:
Is there a reason why CommandGetSpendingByYear doesn't have this try-catch statement? tbh I think having this may be an element of defensive programming but I'm not sure haha. lmk!

Reply:
I have actually made the changes to the CommandGetSpendingByYear in the next PR.